libs/testing/e2e/src/lib/support/addons.ts

```diff
@@ -0,0 +1,50 @@
+import { expect, Locator, Page } from '@playwright/test'
+import { sleep } from './utils'
+
+expect.extend({
+  async toHaveCountGreaterThan(
```

Comment:
We should phase this out, and remove if unused. It's janky in the trace viewer, and can be accomplished in other ways.

Reply:
Addressing in https://github.com/island-is/island.is/pull/16816